components/public-api/gitpod/v1/user.proto

```diff
@@ -0,0 +1,203 @@
+syntax = "proto3";
+
+package gitpod.v1;
+
+import "gitpod/v1/editor.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/v1";
+
+service UserService {
+  // GetAuthenticatedUser allows to retrieve the current user.
+  rpc GetAuthenticatedUser(GetAuthenticatedUserRequest) returns (GetAuthenticatedUserResponse) {}
+}
+
+message GetAuthenticatedUserRequest {}
+message GetAuthenticatedUserResponse {
+  User user = 1;
+}
+
+message User {
+  // id is a UUID of the user
+  string id = 1;
+
+  // organization_id is the id of the organization this account is owned by.
+  //
+  // +optional if not set, this account is owned by the installation.
+  string organization_id = 2;
+
+  // name is the username
+  string name = 3;
+
+  // avatar_url is a link to the user avatar
+  string avatar_url = 4;
+
+  // created_at is the creation time
+  google.protobuf.Timestamp created_at = 5;
+
+  repeated Identity identities = 6;
+
+  // blocked indicates whether the account is blocked, e.g. due to vialoation of ToS.
+  bool blocked = 7;
+
+  // last_verification_time is the last time this account was verified.
+  //
+  // +optional if not set, this account is not verified.
+  google.protobuf.Timestamp last_verification_time = 8;
+
+  // verification_phone_number is the phone number used for the last verification.
+  string verification_phone_number = 9;
+
+  // +optional
+  WorkspaceTimeoutSettings timeout_settings = 10;
+  message WorkspaceTimeoutSettings {
+    // user globol workspace timeout
+    //
+    // +optional
+    string workspace_timeout = 1;
```

Comment:
@AlexTugarev let's align with https://github.com/gitpod-io/gitpod/blob/49433cebe6c3c1d1a660dae9c2980273ca48f2c1/components/public-api/gitpod/v1/workspace.proto#L184-L188

it should be duration and called `inactivity`

Reply:
that's updated now. can we proceed?

Reply:
@akosyakov, where are the translation rules for mapping single `workspaceTimeout` to `[inactivity, disconnected]`? I figured out how to map the current duration strings to `Duration` of protobuf, so that's not the problem.

Reply:
is it the same like https://github.com/gitpod-io/gitpod/pull/19132#discussion_r1405837106?